go/cmd/berty/main.go

```diff
@@ -169,12 +172,42 @@ func main() {
 			// protocol
 			var protocol bertyprotocol.Service
 			{
-				api, node, err := ipfsutil.NewInMemoryCoreAPI(ctx)
+
+				mardv := multiaddr.StringCast(RendezVousServer)
+				rdvpeer, err := peer.AddrInfoFromP2pAddr(mardv)
+				if err != nil {
+					return errcode.TODO.Wrap(err)
+				}
+
+				routingOpts, crouting := ipfsutil.NewTinderRouting(logger, rdvpeer.ID, false)
```

Comment:
not sure about the naming here, is `routingOpts`, actually a real Opts, or more like a `router`?

is `croutine` a chan to now when it's "done"? if yes, maybe something like `routerFinished`

Reply:
- it's an actual routing option in ipfs build config: `libp2p.RoutingOption`
- `crouting` output the actual routing used by ipfs (inside a `routingOut` struct) along the dht instance